<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1470</width>
    <height>1271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>521</width>
      <height>411</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Downloads/Meteor.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocity at Impact</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-99999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999999999.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Meteor Density</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-9999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Primary Impact Lasted</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999.990000000005239</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Answers</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-99999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius of the meteor</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>60</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Average Force Applied During Collision</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>350</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Explanation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>250</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Previous F Average Value</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1470</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
